docs/install/install-server/configure-postgres.inc

```diff
@@ -17,10 +17,11 @@ Initialize the PostgreSQL server:
 
   sudo /usr/pgsql-10/bin/postgresql-10-setup initdb
 
-Start the PostgreSQL database
+Start the PostgreSQL database and make sure it is started at boot.
 
 .. code-block:: sh
 
+   sudo systemctl enable postgresql-10
    sudo systemctl start postgresql-10
```

Comment:
@arnaudsjs what do you think? Use this or:

systemctl enable --now postgresql-10

It is the same but I see it seldom in guides because it is quite unknown. It works on Centos 7

Reply:
I agree that this option isn't well knows, but that's not really a valid argument to not use it. Since the line above clearly mentions what the command does, I would go for: `systemctl enable --now postgresql-10`

Reply:
agree